src/pip/_internal/commands/debug.py

```diff
@@ -63,6 +63,14 @@ def create_vendor_txt_map():
     return dict(line.split('==', 1) for line in lines)  # type: ignore
 
 
+def create_debundle_txt_map():
+    # type: () -> Dict[str, str]
+    wheels = [fn for fn in os.listdir(pip._vendor.WHEEL_DIR)]
```

Comment:
This can also be changed to:

```python
list(*map(os.listdir, pip._vendor.WHEEL_DIR)
```

but if this reduces readability, the original version is fine as well.